scripts/hf_sim.py

```diff
@@ -400,19 +400,20 @@ def run_hf(local_statfile, n_stat, idx_0, velocity_model=args.velocity_model):
         str(args.vs_moho),
         "%d %s %s %s %s %d" % (nl_skip, vp_sig, vsh_sig, rho_sig, qs_sig, ic_flag),
         velocity_name,
+        "%s %s %s" % (args.fa_sig1, args.fa_sig2, args.rv_sig1),
```

Comment:
Doesn't this screw up for previous versions of HF eg. 5.4.5.4

Reply:
well, this is actually what the previous version needed, this line was somehow skipped from the last PR #294 
took me a while to find what was missing.
And this is why i want to keep the original 'template', so in the future if more params were needed, the base will not change.